### PR TITLE
Ready to support parse-server which hosted by ourselves

### PR DESCRIPTION
**Pull request by sdf611097**
https://github.com/ParsePlatform/parse-server
Parse will not host api server after 2017/1/28

Original endpoint 
https://api.parse.com/1
Hosted by ourselves
http://localhost:1337/parse

### CODE REVIEW

File: test/objects.test.js [hunk 1]
```diff
@@ -396,10 +415,10 @@ describe('objects', function() {
 
           var dog = null,
               newBreed = "";
-
+          console.log('body2',body);
```
Comment (shiki) [inline test/objects.test.js]: Would you mind removing this though?

File: test/objects.test.js [hunk 2]
```diff
@@ -330,9 +330,28 @@ describe('objects', function() {
...
+            002 the service is currently unavailable
+            124 timeout
+            154 out of count requestlimit
+            155 out of normal requestlimit
+            159 temporary error
+          */
+          var retryable = [ 2, 124, 154, 155, 159];
+          if(body.code && retryable.indexOf(body.code) > -1){
+            console.log('code', body.code);
+            console.log('2', 'the service is currently unavailable');
```
Comment (shiki) [inline test/objects.test.js]: Sorry I could not quite figure out what this is trying to test. Would you please explain? :)

File: test/objects.test.js (hunk 1 above)
Comment (sdf611097) [inline test/objects.test.js]: Sorry remove it, please.
I have found that, but git commit --amend seems only works in local.

File: test/objects.test.js (hunk 2 above)
Comment (sdf611097) [inline test/objects.test.js]: If the query format is wrong, you may get something like{"code":107,"error":"cannot route batch path /classes/GameScore"} .

In this case err is undefined(or null?). Parse Error object is set in body with code and error.
2, 124, 154, 155, 159 is something wrong or be rejected on parse hosted server.

**Comment by sdf611097** [comment]
BTW. I have a commit to fixe file.test.js to pass the test. Should I do a new pull request?

**Pull request closed without merging**